packages/store/addon/-private/identifiers/utils/uuid-v4.ts

```diff
@@ -0,0 +1,67 @@
+// support IE11
+declare global {
+  interface Window {
+    msCrypto: Crypto;
+  }
+}
+
+const CRYPTO =
+  typeof window !== 'undefined' &&
+  window.msCrypto &&
+  typeof window.msCrypto.getRandomValues === 'function'
+    ? window.msCrypto
+    : crypto;
+
+// we might be able to optimize this by requesting more bytes than we need at a time
+function rng() {
+  // WHATWG crypto RNG - http://wiki.whatwg.org/wiki/Crypto
```

Comment:
Is the crypto one just as fast? We don't need crypto strength

Reply:
if we generate a uuid then we must use the crypto lib as it is the only thing that gives us access to generation of random bytes, which is necessary for generating true uuid's